### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by janfaracik**
Happy to add more info if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
-# Jenkins UI sample plugin
+![Jenkins Design Library](logo-dark.svg#gh-dark-mode-only)
```
Comment (timja) [inline README.md:1]: This seems to be linking to https://github.com/jenkinsci/ui-samples-plugin/blob/a4dc40e5c59e22e675f897e7ec6747a8d2633970/logo-dark.svg#gh-dark-mode-only

When I click it, is there a way to make it not a link?

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
FYI I added you to @jenkinsci/sig-ux and made it admin of this repo.
I forgot that team existed

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
-This fake plugin shows Jenkins developers how to build user interfaces using Jenkins UI components.
+Design Library makes it easy for developers to build complex and consistent interfaces using Jenkins UI components
 
 This plugin contains:
-- demonstration of UI controls available in Jenkins based on Stapler, Jelly, Groovy, etc.
-- snippets of Jelly/Groovy "ready-to-use" examples
+- demonstration of UI components available in Jenkins
+- snippets of Jelly ready-to-use examples
 
 ## Usage
```
Comment (NotMyFault) [inline README.md:10]: We could add a section with additional information like linking to the jelly tag reference used in core and a link to the gitter room for ux-sig where people could head to for additional information.

**Review by NotMyFault — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (timja) [reply README.md:11]: I think that should belong in the contributing document see https://github.com/jenkinsci/ui-samples-plugin/issues/16

**Comment by timja** [comment]
@NotMyFault did you want to join the ux-sig team?

File: README.md (hunk 1 above)
Comment (janfaracik) [reply README.md:1]: Had a Google but nothing helpful comes up sadly, there's a hacky way to link it to `#` so it won't navigate to another page but it isn't ideal.

File: README.md (hunk 2 above)
Comment (NotMyFault) [reply README.md:10]: Oic, I didn't see your issue here.

File: README.md (hunk 1 above)
Comment (timja) [reply README.md:1]: That’s fine then

**Comment by NotMyFault** [comment]
> @NotMyFault did you want to join the ux-sig team?

For sure, that would be awesome!

**Comment by timja** [comment]
Added you to the team, tick is now green

**Pull request merged**